easybuild/easyconfigs/s/SPAdes/spades.patch

```diff
@@ -0,0 +1,58 @@
+* #import is unknown pragma for icpc, but apparently it can be removed without problems...
+* translating 'a < b' to 'b > a' is required to dance around weird compiler error: error: expected a ">"
+* __is_trivial define is required to dance around bug in Intel compilers (intel/2016b), see https://software.intel.com/en-us/forums/intel-c-compiler/topic/685388
+* cast to bool is required because Intel compilers complaining there's no implicit conversion to bool when Boost 1.63.0 is used as dependency
+--- SPAdes-3.10.1/src/common/modules/path_extend/ideal_pair_info.hpp.orig	2017-03-02 20:23:08.979343521 +0100
++++ SPAdes-3.10.1/src/common/modules/path_extend/ideal_pair_info.hpp	2017-03-02 20:23:26.519525771 +0100
+@@ -14,7 +14,6 @@
+ 
+ #ifndef IDEAL_PAIR_INFO_HPP_
+ #define IDEAL_PAIR_INFO_HPP_
+-#import <vector>
+ #include "pipeline/graph_pack.hpp"
+ 
+ namespace path_extend {
+--- SPAdes-3.10.1/src/common/assembly_graph/dijkstra/dijkstra_algorithm.hpp.orig	2017-02-28 15:55:35.000000000 +0100
++++ SPAdes-3.10.1/src/common/assembly_graph/dijkstra/dijkstra_algorithm.hpp	2017-03-03 10:21:48.506197733 +0100
+@@ -39,7 +39,7 @@
+ 
+   bool operator()(T obj1, T obj2){
+       if(obj1.distance != obj2.distance)
+-          return obj2.distance < obj1.distance;
++          return obj1.distance > obj2.distance;
```

Comment:
without this change, the Intel compilers complain with `error: expected a ">"`, which is just weeeeeiiiiird...

Same problems with more recent versions of Intel compilers (incl. 2017 update 2)